### PR TITLE
Feature/ml #16 registration page

### LINKED ISSUE #15: As a user I want to set up my profile data
As a user I want to set up my profile data

- [x] Write components for page (MainInfo, AdditionalInfo, LanguagesInfo, PreviewProfile)

- [x] Fetch data from backend and display on the fields 

- [x] Change data from fields and save them sending to backend by clicking on button Save

- [x] Discard all changes by clicking on button Cancel

- [x] Open a dropdown menu by clicking fileds: Birthday, Country, I speak, Level

- [x] Upload a photo by clicking on image Upload photo

- [x] Add new language by clicking on button [+ add language]

- [x] collect data from filled fields and display on PreviewInfo Component

- [x] Add navigation to buttons : Cancel and Save

### LINKED ISSUE #16: As a user I want to sign up and sign in
As a user I want to sign up and sign in 

- [x] Write components for page (MainInfo, AdditionalInfo, LanguagesInfo, PreviewProfile)

- [x] Do a check on validation of the fields( Filled or not)

- [x] Open a dropdown menu by clicking fileds: Birthday, Country, I speak, Level

- [x] Upload a photo by clicking on image Upload photo

- [x] Add new language by clicking on button [+ add language]

- [x] collect data from filled fields and display on PreviewInfo Component

- [x] Add navigation to buttons : Cancel and Sign Up

### PR DESCRIPTION
**Pull request by romanazaruk**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -12,6 +13,7 @@
     "@types/react-dom": "^16.9.8",
     "@types/react-router-dom": "^5.1.6",
     "react": "^16.13.1",
+    "react-datepicker": "^3.3.0",
```
Comment (oleksandrstarov) [inline frontend/package.json]: Where this import is used in the code?

File: frontend/src/components/pages/login/login.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    .logo {
+      margin: 0 $gap-10 0 -16px;
+    }
+
+    .logo-wrapper {
+      display: flex;
+
+      h1 {
+        font-size: 38px;
```
Comment (oleksandrstarov) [inline frontend/src/components/pages/login/login.scss:26]: such params should be global

File: frontend/src/components/shared/additionalInfo/additionalInfo.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
+@import '../../../styles/variables.scss';
+
+p {
+  margin: 24px 0 10px;
+}
```
Comment (oleksandrstarov) [inline frontend/src/components/shared/additionalInfo/additionalInfo.scss]: this duplicates few times - maybe it is better to add class name and use as global style?

File: frontend/src/components/shared/languagesInfo/Languages.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import LanguagesDetailInfo from './components/LanguagesDetailInfo';
+
+const LanguagesInfo: React.FC = () => {
+  return (
+    <Tile>
+      <Box boxShadow={2} className="tile">
+        <h1>Languages</h1>
+        <div className="languages-div">
+          <LanguagesDetailInfo do="I speak" level="Level" />
+          <div className="divider" />
```
Comment (oleksandrstarov) [inline frontend/src/components/shared/languagesInfo/Languages.tsx]: is there possible to use hr tag instead?

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: frontend/src/components/shared/button/Button.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import { Link } from 'react-router-dom';
+
+type ButtonProps = {
+  name: string;
+  style: string;
+};
+
+const Button: React.FC<ButtonProps> = props => {
+  return (
+    <Link to="/login" className={props.style}>
```
Comment (apastuhov) [inline frontend/src/components/shared/button/Button.tsx]: shared and abstract button which will always point to login? maybe it is better to rename it or make more abstract, also it is not a button, but Link which Maybe looks like a button and depend on props.

**Review by apastuhov — CHANGES_REQUESTED** [review]

**Pull request merged**